backend/config/sentry.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
I think it would be simpler to have everything in only one file. Just store this under a `sentry` key in the main config?

Reply:
The main configuration file should not be imported in a public file otherwise all our secrets will be made public.

Reply:
They will be made public if someone does `window.config = ...` but otherwise it won't, no? 
Everything is wrapped in anonymous functions with Webpack

Reply:
Everything in `backend/config/sentry.js` is made available from the `dist` folder. If I import the main configuration file (which contains a Sentry DSN with a secret) that secret will be made public.

I may not understand your proposal correctly. Let's chat away from keyboard if that's the case. :smiley: